### PR TITLE
Multitool CBM gets Cutting 2, extended descriptor qualities

### LINKED ISSUE #53123: Magus Battle Staff Requires a Nonexistent Tool
### Describe the bug

Magiclysm's magus battle staff requires a tool with cutting 2 which does not exist.

### Steps to reproduce

1. Enter game
2. Look at recipe
3. Sadness

### Expected behavior

Being able to craft the Staff

### Screenshots

_No response_

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: fa91686 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions],
    C.R.I.T Expansion Mod [crt_expansion],
    Magiclysm [magiclysm],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    No Fungal Monsters [No_Fungi]
]

### Additional context

_No response_

### LINKED ISSUE #53230: Intergrated toolset not displaying tool quality
### Describe the bug

Intergrated toolset not displaying its tool qualities

### Steps to reproduce

1. Install intergrated toolset CBM
2. Wield the extended multitool
3. Check the extended multitool you're wielding
4. Displays no tool qualities 

### Expected behavior

It should display it's qualities 

### Screenshots

_No response_

### Versions and configuration

Version: c232b41

### Additional context

_No response_

### LINKED ISSUE #53315: Combat and Motorcycle boots are no longer able to be disassembled due to #52889
### Describe the bug

Issue #52889 reworked integrated toolset to integrated welder, removing a large number of tool qualities from it.
Issue #53197 revamped it slightly, adding a handful of qualities back, but not Cutting 2

This has now rendered Combat and Motorcycle Boots impossible to deconstruct, with two side effects:
1. Neither recipe can be learned, as they're only learned through deconstruction.
2. Kevlar Thread has effectively become unobtanium, given the only source available now is a 20% chance for the Exodii merchant to have _at most 6_ of them.

#53059 is going to audit disassembly and materials, but this is a different issue.
More info below.

### Steps to reproduce

1. I acquire a pair of combat or motorcycle boots
2. I attempt to disassemble combat or motorcycle boots
3. I am informed that they cannot be disassembled as the disassembly required Cutting 2

### Expected behavior

1. I acquire a pair of combat or motorcycle boots
2. I attempt to disassemble combat or motorcycle boots
3. I receive the components from disassembly.

### Screenshots

Here is the combat boots recipe from in game:
2021-12-09 10_35_53-Cataclysm_ Dark Days Ahead - 66c53a1

### Versions and configuration

OS: Windows 11
Game version: 0.F-4306-g61fa86a117 built from source
Graphics version: Tiles
in game language: English
Mods: dda, disable npc needs, no fungal growth

### Additional context

after running through the code in debug, it's an issue within src/requirements.cpp, starting around line 1169 with the emplace_back method. When Sewing is replaced with Cut, it's passing (qual_CUT, 1, quality.level). The quality level is the quality of the sewing tool requirement - level 2. 

Combat boots and Motorcycle boots are the only two Sewing 2 items that can be deconstructed - which is why this may have been missed.

Additionally, sewing levels 3 and 4 (sewing and tailor's kits, respectively) are actually unused across all recipes.

### PR DESCRIPTION
**Pull request by anoobindisguise**
#### Summary
None

#### Purpose of change
Fix #53230 fix #53123 fix #53315

#### Describe the solution
Restore cutting 2 to multitool/toolset CBM. Also restore description to extended tool. Cutting 2 apparently actually matters for some specific cases so I am restoring it to the tool. 

#### Describe alternatives you've considered
Wait for the cutting 1/cutting 2 weirdness to be fixed.

### CODE REVIEW

**Comment by catdach** [comment]
It is really weird that no other tools in the game have cutting 2. Even a scalpel is just cutting 1. Kinda calls in to question why cutting and fine cutting are even separate qualities.

File: data/json/items/fake.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
       [ "WRENCH_FINE", 1 ],
       [ "SCREW", 1 ],
       [ "SCREW_FINE", 1 ],
-      [ "CUT", 1 ],
+      [ "CUT", 2 ],
```
Comment (kevingranade) [inline data/json/items/fake.json:42]:
```suggestion
      [ "CUT", 1 ],
```

File: data/json/items/tool/workshop.json [hunk 2]
```diff
@@ -1119,6 +1119,19 @@
...
+    ],
+    "qualities": [
+      [ "SAW_W", 1 ],
+      [ "SAW_M", 1 ],
+      [ "SAW_M_FINE", 1 ],
+      [ "WRENCH", 1 ],
+      [ "WRENCH_FINE", 1 ],
+      [ "SCREW", 1 ],
+      [ "SCREW_FINE", 1 ],
+      [ "CUT", 2 ],
```
Comment (kevingranade) [inline data/json/items/tool/workshop.json:1131]:
```suggestion
      [ "CUT", 1 ],
```

**Comment by kevingranade** [comment]
Giving this CBM exceptional cutting ability isn't the answer here, it's either adding exceptional cutting quality to other tools such as the proposed kevlar shears, or removing the requirement for it.
I'm leaning toward the kevlar shears thing.

**Pull request merged**